doghouse/server/doghouse.go

```diff
@@ -3,49 +3,35 @@ package server
 import (
 	"bytes"
 	"context"
+	"errors"
 	"fmt"
-	"net/http"
-	"strings"
-	"time"
 
 	"github.com/google/go-github/v60/github"
 
+	"github.com/reviewdog/reviewdog"
 	"github.com/reviewdog/reviewdog/diff"
 	"github.com/reviewdog/reviewdog/doghouse"
 	"github.com/reviewdog/reviewdog/filter"
 	"github.com/reviewdog/reviewdog/proto/rdf"
-	"github.com/reviewdog/reviewdog/service/github/githubutils"
+	ghService "github.com/reviewdog/reviewdog/service/github"
 )
 
-// GitHub check runs API cannot handle too large requests.
-// Set max number of filtered findings to be shown in check-run summary.
-// ERROR:
-//
-//	https://api.github.com/repos/easymotion/vim-easymotion/check-runs: 422
-//	Invalid request.
-//	Only 65535 characters are allowed; 250684 were supplied. []
-const maxAllowedSize = 65535
-
-// > The Checks API limits the number of annotations to a maximum of 50 per API
-// > request.
-// https://developer.github.com/v3/checks/runs/#output-object
-const maxAnnotationsPerRequest = 50
-
 type Checker struct {
 	req              *doghouse.CheckRequest
-	gh               checkerGitHubClientInterface
+	cli              *github.Client
 	inDogHouseServer bool // If true, this checker runs in the DogHouse server.
 }
 
 func NewChecker(req *doghouse.CheckRequest, gh *github.Client, inDogHouseServer bool) *Checker {
 	return &Checker{
 		req:              req,
-		gh:               &checkerGitHubClient{Client: gh},
+		cli:              gh,
 		inDogHouseServer: inDogHouseServer,
 	}
 }
 
-func (ch *Checker) Check(ctx context.Context, makeRequest bool) (*doghouse.CheckResponse, error) {
+func (ch *Checker) Check(ctx context.Context) (*doghouse.CheckResponse, error) {
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>exported method Checker.Check should have comment or be unexported
<!-- __reviewdog__:Cg8zMzk1OGY1YjFiYzMwMWUSD2dvbGludC1jaXJjbGVjaQ== -->